subsys/net/lib/aws_fota/include/aws_fota_json.h

```diff
@@ -27,10 +27,36 @@ extern "C" {
  */
 #define STATUS_MAX_LEN (12)
 
-int aws_fota_parse_notify_next_document(char *job_document,
-		u32_t payload_len, char *job_id_buf, char *hostname_buf,
-		char *file_path_buf);
+/**
+ * @brief Parse a given AWS IoT DescribeJobExecution response JSON object.
+ *	  More information on this object can be found at https://docs.aws.amazon.com/iot/latest/developerguide/jobs-api.html#mqtt-describejobexecution
+ *
+ * @param[in] job_document JSON formated string of an AWS Job execution
```

Comment:
Nit: IMO it's better having two spaces after the arg name to improve readability.

Reply:
Added